hooks.js

```diff
@@ -30,7 +30,7 @@ function getUserId () {
 
 const proto = Mongo.Collection.prototype;
 const directEnv = new Meteor.EnvironmentVariable(false);
-const methods = ['find', 'findOne', 'insert', 'update', 'remove', 'upsert'];
+const methods = ['find', 'findOne', 'findOneAsync', 'insert', 'insertAsync', 'updateAsync', 'removeAsync', 'upsertAsync'];
```

Comment:
@wildhart findOne and insert is still here (because they are used on client side), isn't it security issue?